### PR TITLE
update leverage marks

### PR DESCRIPTION
**Pull request by vipineth**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206092706230183

### CODE REVIEW

File: src/components/LeverageSlider/LeverageSlider.tsx [hunk 1]
```diff
@@ -1,30 +1,62 @@
...
+    [onChange, keyValueMap]
+  );
+
+  useEffect(() => {
+    if (value !== defaultValue) {
+      handleChange(defaultValue);
+    }
+  }, [value, defaultValue, handleChange]);
+
+  const customHandle = (props: any) => <LeverageSliderHandle {...props} keyValueMap={keyValueMap} />;
```
Comment (onemicky) [inline src/components/LeverageSlider/LeverageSlider.tsx:58]: Just a minor rerender optimization idea: what do you think about wrapping the customHandle into a useMemo hook?

File: src/components/Synthetics/TradeBox/TradeBox.tsx [hunk 2]
```diff
@@ -1245,7 +1245,12 @@ export function TradeBox(p: Props) {
             </ToggleSwitch>
 
             {isLeverageEnabled && (
-              <LeverageSlider value={leverageOption} onChange={setLeverageOption} isPositive={isLong} />
+              <LeverageSlider
+                marks={[0.1, 1, 2, 5, 10, 15, 20, 25, 30, 40, 50]}
```
Comment (onemicky) [inline src/components/Synthetics/TradeBox/TradeBox.tsx:1249]: Another small suggestion on the same field: would you like to consider moving the array to the constant out of the component body?

**Review by onemicky — APPROVED** [review]

File: src/components/Synthetics/TradeBox/TradeBox.tsx (hunk 2 above)
Comment (vipineth) [reply src/components/Synthetics/TradeBox/TradeBox.tsx]: Thanks, updated!

File: src/components/LeverageSlider/LeverageSlider.tsx (hunk 1 above)
Comment (vipineth) [reply src/components/LeverageSlider/LeverageSlider.tsx]: Thanks, updated!

**Review by onemicky — APPROVED** [review]

**Review by onemicky — APPROVED** [review]

**Pull request merged**